src/python/pants/option/global_options.py

```diff
@@ -211,6 +211,12 @@ def register_bootstrap_options(cls, register):
                   'options and specifying the targets directly. These three types of target '
                   'selection are mutually exclusive.')
 
+    register('--query', type=list, default=[], fromfile=True, metavar='<query-expr>',
+             # TODO: what does this option do????
```

Comment:
When true, it causes a daemon client to restart the daemon if the fingerprint of the option has changed. `daemon=False` is correct here.